src/components/patients/index.tsx

```diff
@@ -2,19 +2,55 @@ import React from "react";
 import Header from "components/header";
 import PatientTable from "components/patients/patientTable";
 import SearchTool from "components/patients/searchTool";
+import { getPatients, getPatientsPerQuery } from "services/api";
+import { PatientBundle } from "types";
 import { Card, Elevation } from "@blueprintjs/core";
+import { requestNextPatients } from "services/api";
```

Comment:
You could put this with the import line 5

Reply:
done